src/components/partition-map/partition-map.component.ts

```diff
@@ -385,6 +385,10 @@ export class PartitionMapComponent implements OnInit, OnDestroy {
         return context;
     }
 
+    trackByFn(_index: number, item: any): number {
```

Comment:
`any` should never be used as a type. The type can be determined by inspecting the type of `_segments`.